components/profile_popover/profile_popover.jsx

```diff
@@ -323,6 +330,24 @@ class ProfilePopover extends React.Component {
             );
         }
 
+        if (this.props.enableTimezone) {
+            const currentTimezone = getCurrentTimezone(this.props.user.timezone);
+            if (currentTimezone) {
+                const useMilitaryTime = PreferenceStore.getBool(Constants.Preferences.CATEGORY_DISPLAY_SETTINGS, 'use_military_time');
```

Comment:
Probably is better option to receive this from the redux connect as a property.

Reply:
👍